invenio_records_resources/services/files/service.py

```diff
@@ -255,12 +257,17 @@ def set_file_content(
 
         except FailedFileUploadException as e:
             file = e.file
+            current_app.logger.exception(f"File upload failed: {str(e)}")
```

Comment:
```suggestion
            current_app.logger.exception("File upload transfer failed.")
```

For better Sentry grouping. We'll see the exception anyways there :)